### PR TITLE
Update api-spec.md

### PR DESCRIPTION
**Pull request by markChartIQ**
Proposed Change - update 'OpenFin' specific language in documentation to 'Vendor' where applicable

### CODE REVIEW

**Comment by finos-admin** [comment]
Thank you for your contribution and **Welcome to our Open Source Community!**

To make sure your pull request is accepted successfully, we ask all our open source contributors to sign a [Contributor License Agreement](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement).

Having reviewed our contributor list, we require a CLA for the following people : (@markChartIQ).

If you need help obtaining a CLA, please read the [Requirements for Contributions](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement) section of our CLA wiki or email [help@finos.org](mailto:help@finos.org) with your questions.

Thanks once again for your contribution. Let us work with you to make the CLA process quick, easy and efficient so we can move forward with reviewing and accepting your pull request.

_cc_ **@finos-admin**

File: docs/api/api-spec.md [hunk 1]
```diff
@@ -114,9 +114,9 @@ There are a wide range of workflows where decoupled intents and/or context passi
 
 ```js
 const chart = await fdc3.raiseIntent('ViewChart');
-// construct an OpenFin wrapper for the App
+// construct a vendor specific wrapper for the App
```
Comment (rikoe) [inline docs/api/api-spec.md]: `vendor-specific` (with a hyphen)?

**Comment by rikoe** [comment]
@markChartIQ - for code changes, you need either an individual or corporate contributor's license agreement with your GitHub account added to it. Since this is docs, not code, we will ignore the message, but probably a good idea to kick this process off anyway?

**Comment by rikoe** [comment]
@markChartIQ - can you please rebase your branch to pull in the latest changes in master, as there are some conflicts?

**Pull request closed without merging**